tests/FsUnit.Xunit.Test/typed.shouldHaveLengthTests.fs

```diff
@@ -3,7 +3,8 @@ namespace FsUnit.Typed.Test
 open Xunit
 open FsUnitTyped
 
-type ``haveLength tests``() =
+type ``shouldHaveLength tests``() =
+
```

Comment:
all files have this new line after type, is it added by Fantomas? If yes, I do not see Fantomas config changes in this PR...

Reply:
I added that manually for better readability.